action.yaml

```diff
@@ -0,0 +1,29 @@
+name: 'Iter8 Install'
+description: 'Install Iter8 CLI'
+inputs:
+  version:
```

Comment:
Could we please get rid of the `version` input?
***
```yaml
- uses: iter8-tools/iter8@v0.10.5
```

or 

```
- uses: iter8-tools/iter8@master
```
are the two types of usage we want to support. In the former case, the version info is explicitly supplied. In the latter case, we want to use your logic for figuring out the latest release, and install it. In both cases, you can extract the [`github.action_ref`](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context), and do not required a version to be supplied as a separate input.

Reply:
The context variable `github.action_ref` is always the empty string in the `run`. https://github.community/t/getting-information-about-my-action/231562 identifies a work around by reading the context variable in the step `env`. In this way we are able to implement this.

In particular, 

- `@vMAJAOR.MINOR.PATCH` will use the matching version
- `@vMAJOR.MINOR` will identify the latest release matching MAJOR and MINOR
- `@stable` will identify the latest release (is this a suitable release name? feel that `@master` or `@latest` will cause misunderstandings)